closure/goog/ui/datepicker.js

```diff
@@ -343,6 +352,33 @@ goog.ui.DatePicker.Events = {
 goog.ui.DatePicker.prototype.isCreated =
     goog.ui.DatePicker.prototype.isInDocument;
 
+/**
+ * @return {number} The timezone offset from UTC in minutes.
+ */
+goog.ui.DatePicker.prototype.getTimezoneOffset = function() {
+  return this.timezoneOffset_;
+};
+
+/**
+ * @return {number} The timezone offset from UTC in milliseconds.
+ * @private
+ */
+goog.ui.DatePicker.prototype.getTimezoneOffsetMs_ = function() {
+  return this.timezoneOffset_ * 60 * 1000;
+};
+
+/**
+* @return {goog.date.Date} The Date object for today.
+* @private
+*/
+goog.ui.DatePicker.prototype.getToday_ = function() {
+  if (goog.isNull(this.timezoneOffset_)) {
+    return new goog.date.Date();
+  }
+  var date = new goog.date.UtcDateTime();
```

Comment:
Simply use `Date` instead of `goog.date.UtcDateTime`  because you don't use any of the features that `goog.date.UtcDateTime` provides to you and when `goog.ui.DatePicker.prototype.setDate` will be called it'll build a new `goog.date.Date` anyways and furthermore it does already accept a `Date` too.
Also since `goog.date.Date` mimics `Date` so this won't be an issue in `goog.ui.DatePicker.prototype.redrawCalendarGrid_` either.
The only reason you can't use `goog.date.Date` directly is that it strips the time information.